root/io/uniquePointer/CMakeLists.txt

```diff
@@ -65,6 +65,26 @@ ROOTTEST_ADD_TEST(simpleRead
                   OUTREF simpleRead.ref
                   DEPENDS simpleWriteRead)
 
+ROOTTEST_ADD_TEST(privacy-dict
+                  COMMAND ${ROOT_rootcling_CMD} -f privacy.dict.cc
+                          ${CMAKE_CURRENT_SOURCE_DIR}/privacy.hh
+                          ${CMAKE_CURRENT_SOURCE_DIR}/privacyLinkDef.h
+                  OUTREF privacyDict.oref
+                  ERRREF privacyDict.eref)
+
+ROOTTEST_GENERATE_DICTIONARY(privacy privacy.hh LINKDEF privacyLinkDef.h)
```

Comment:
It's not clear to me what is the difference between these two. Isn't also the first one generating the dictionary for the `privacy` header?

Reply:
The 2nd also generates a dictionary.  I used it here not introduce a special rules/way to generate a dictionary based library.  The previous test main purpose is to capture and check the `stdout` and `stderr` content.